param.ini

```diff
@@ -0,0 +1,34 @@
+[integration]
```

Comment:
Maybe also add a file path on where to save the data? with standard being the usual map

Reply:
Added the option